core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/LibraryPropagationRule.java

```diff
@@ -106,4 +106,8 @@ public static boolean validate(Library library, Iterable<LibraryPropagationRule>
     return first;
   }
 
+  public boolean isDesign() {
+    return name != null && !name.isEmpty() && platformName != null && paired != null && librarySelectionType != null
+        && libraryStrategyType != null;
+  }
```

Comment:
I will add a comment. A library design is one that has no wildcards in it. So, if you allow any selection strategy, they you can't be a design. So long as everything is locked, a rule is a design.